ui/pages/swaps/awaiting-swap/awaiting-swap.js

```diff
@@ -230,6 +236,22 @@ export default function AwaitingSwap({
     );
   }
 
+  const MakeAnotherSwap = () => {
+    return (
+      <Box marginBottom={3}>
+        <a
```

Comment:
I think it would be best to use a `<button>` here

Reply:
Thanks for the suggestion! We had a quick chat about when to use a button or a link and since we do page redirection on click here, a link seems fine.